docker-compose.yml

```diff
@@ -11,8 +11,10 @@ services:
             - python
             - python-sync
             - redis
+            - cherami
         environment:
-            - WAIT_FOR=go,node,python,python-sync
+            - WAIT_FOR=go,node,python,python-sync,cherami
```

Comment:
That's pretty significant - wondering if we can get that down to < 1s, perhaps using a pre-burned Docker image...